src/App.css

```diff
@@ -1,39 +1,151 @@
-.App {
-  text-align: center;
+:root {
+  --red: #9353a1;
+  --light-red: #c27bc1;
+  --light-bg: #bfa3e3;
+  --white: white;
+  --active: yellow;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: var(--light-bg);
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+li {
+  list-style-type: none;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+a {
+  text-decoration: none;
+  font-size: 1.2rem;
+  color: var(--white);
+}
+
+.logo {
+  border: 1px solid var(--white);
+  padding: 10px 20px;
+}
+
+.d-flex-space-be {
   display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+header {
+  background-color: var(--red);
+  height: 80px;
+  padding-inline: 10px;
+}
+
+nav {
+  height: 100%;
+  align-items: center;
+}
+
+.card {
+  padding: 50px;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  gap: 20px;
+  background-color: var(--red);
+  color: var(--white);
+  cursor: pointer;
+  outline: none;
+  border: none;
+}
+
+.card:nth-child(4n + 2),
+.card:nth-child(4n + 3) {
+  background-color: var(--light-red);
+}
+
+.py5 {
+  padding-block: 10px;
 }
 
-.App-link {
-  color: #61dafb;
+.grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
+  justify-content: flex-start;
+  padding: 10px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+.my5 {
+  margin-top: 10px;
+}
+
+.search-container {
+  width: 100%;
+  padding-inline: 10px;
+}
+
+.search {
+  width: 100%;
+  line-height: 2.5;
+  padding: 5px 10px;
+  border-radius: 5px;
+  font-family: inherit;
+  font-size: 1rem;
+  outline: none;
+  border: 3px solid var(--red);
+  box-shadow:
+    rgba(0, 0, 0, 0.1) 0 10px 15px -3px,
+    rgba(0, 0, 0, 0.05) 0 4px 6px -2px;
+}
+
+form {
+  position: relative;
+}
+
+.search-icon {
+  position: absolute;
+  top: 20px;
+  right: 3rem;
+  cursor: pointer;
+}
+
+.detail-title {
+  text-transform: uppercase;
+  text-align: center;
+}
+
+.details {
+  color: var(--white);
+  max-width: 500px;
+  margin: 10px auto;
+  border: 2px solid red;
+  padding: 10px;
+  border-radius: 5px;
+}
+
+.details > * {
+  margin-block: 10px;
+}
+
+.details-body > * {
+  margin-block: 10px;
+  font-size: 1.2rem;
+  font-weight: 700;
+  padding: 10px;
+}
+
+.details-body > *:nth-child(even) {
+  background-color: var(--light-red);
+}
 
-  to {
-    transform: rotate(360deg);
-  }
+.back-arrow {
+  font-size: 2rem;
+  cursor: pointer;
 }
```

Comment:
- [ ]  I suggest that you make sure that you adjust the styling ( layout, spacing, margin, element alignment i.e the alignment of the `BITCOIN BTC` text) match the Design guidelines :pray: Ensure that your back icon is placed on the left side on the navigation bar as it is in the design. Also on the given design, there is no red border on the detail page so make sure that you remove the red border you have here :point_down: 

  | Your Design | Design guidelines |
  | :---:   | :---: | 
  | ![image](https://github.com/DebyGrey/metric-app/assets/92303891/91288148-19f4-480c-9bff-e0d03537b915)| ![image](https://github.com/DebyGrey/metric-app/assets/92303891/ab0b8ddd-c357-461b-9af3-4b21ddb5b493)|